screens/pages/components/LitterBottomSearch.js

```diff
@@ -118,6 +120,9 @@ class LitterBottomSearch extends PureComponent {
                 {
                     text: 'OK',
                     onPress: () => {
+                        this.props.swiperIndexChanged(
+                            currentIndex > 0 ? currentIndex - 1 : 0
+                        );
```

Comment:
Why do we need to change the index here?

Reply:
To keep state in sync and remove any possibility of encountering "index out of bounds" error.

Reply:
I see. Can you wrap the condition in brackets so it becomes `(currentIndex > 0)` ?